terms/ResponseOption

```diff
@@ -0,0 +1,10 @@
+{
+	"@context": {
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#"
+    },
+    "@id": "ResponseOptions",
```

Comment:
id and term filename are different. and i thought ResponseOption is a schema.

Reply:
sorry, this one i committed by mistake. it is a schema. I'll delete this file.